Source/Renderer/Framebuffer.js

```diff
@@ -308,6 +308,11 @@ define([
         gl.bindFramebuffer(gl.FRAMEBUFFER, null);
     };
 
+    Framebuffer.prototype._attachTexture = function(context, attachment, texture) {
+        context.bindFramebuffer(this);
```

Comment:
It would be cleaner just to call `_bind` here, but then you end up re-binding the FBO when you render the shadow casters.